src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigCol {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, Color color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double getFirstLeg(double firstLeg) {
+        return firstLeg;
+    }
+
+    public double getSecondLeg(double secondLegLeg) {
+        return secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: " + getArea()
+                + " sq. units, firstLeg: "
+                + firstLeg + " units, secondLeg: " + secondLeg + " units, color: " + getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
```

Comment:
Avoid using magic numbers directly in the code. It's better to declare them as named constants to improve code readability and maintainability.